sceptre/config/reader.py

```diff
@@ -245,8 +254,9 @@ def read(self, rel_path, base_config=None):
     def _recursive_read(self, directory_path, filename):
         """
         Traverses the directory_path, from top to bottom, reading in all
-        relevant config files. If config items appear in files lower down the
-        stack_group tree, they overwrite items from further up.
+        relevant config files. If config attributes are encountered further
+        down the StackGroup they are merged with the parent as defined in the
+        `CONFIG_MERGE_STRATEGIES` dict.
 
         :param directory_path: Relative directory path to config to read.
         :type directory_path: str
```

Comment:
Not this line, but the description for `filename` (or its name) seem out of whack.